mmv1/third_party/terraform/tests/resource_workstations_workstation_test.go.erb

```diff
@@ -0,0 +1,88 @@
+<% autogen_exception -%>
+package google
+<% unless version == "ga" -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccWorkstationsWorkstation_basic(t *testing.T) {
```

Comment:
I think this test case as written largely duplicates `mmv1/templates/terraform/examples/workstation_basic.tf.erb`? We'll want to add a handwritten update test, to exercise PATCH for the resource, though.

Reply:
I've changed this into an update test, where we update the `display_name` 👍🏼